cpp/test/linalg/divide.cu

```diff
@@ -59,6 +59,7 @@ class DivideTest : public ::testing::TestWithParam<raft::linalg::UnaryOpInputs<T
     int len = params.len;
     RAFT_CUDA_TRY(cudaStreamCreate(&stream));
```

Comment:
I've been removing this logic in the other tests as I've been encountering it. I believe this pattern predates the time when the primitives were made aware of the handle. It looks like the handle is already being used in the gtest class. Instead of creating a new stream and having to synchronize it, can we just always use the stream from the handle?